src/Core/Migration/Migration1754897550AddFieldsToMigrationLogs.php

```diff
@@ -69,31 +82,20 @@ public function update(Connection $connection): void
         $this->dropIndexIfExists($connection, self::MIGRATION_LOGGING_TABLE, 'idx.swag_migration_logging.run_id_code');
         $this->dropConstraintIfExists($connection, 'json.swag_migration_logging.log_entry');
 
-        $this->dropObsoleteColumns($connection, $schemaManager);
+        $this->dropObsoleteColumns($connection);
         $this->addOrModifyColumns($connection, $schemaManager);
         $this->ensureRelations($connection, $schemaManager);
     }
 
-    /**
-     * @param AbstractSchemaManager<MySQLPlatform> $schemaManager
-     */
-    private function dropObsoleteColumns(Connection $connection, AbstractSchemaManager $schemaManager): void
+    private function dropObsoleteColumns(Connection $connection): void
     {
-        $columns = $schemaManager->listTableColumns(self::MIGRATION_LOGGING_TABLE);
-
-        $allFields = array_merge(
-            self::REQUIRED_FIELDS,
-            self::OPTIONAL_FIELDS,
-            self::SYSTEM_FIELDS
-        );
-
-        foreach ($columns as $column) {
-            if (!\array_key_exists($column->getName(), $allFields)) {
+        foreach (self::FIELDS_TO_DROP as $column) {
+            if ($this->columnExists($connection, 'swag_migration_logging', $column)) {
                 $connection->executeStatement(
                     \sprintf(
                         'ALTER TABLE `%s` DROP COLUMN `%s`;',
                         self::MIGRATION_LOGGING_TABLE,
-                        $column->getName()
+                        $column
                     )
                 );
             }
```

Comment:
While you are touching this, would be great if this would just use the existing method:
https://github.com/shopware/shopware/blob/9f97e99531f360d220a3a667e141a9cd0a00bcac/src/Core/Framework/Migration/MigrationStep.php#L100-L103